openlibrary/templates/lib/header_dropdown.html

```diff
@@ -21,6 +23,8 @@
       <summary $:cond(props['name'] == 'hamburger', 'data-ol-link-track="HeaderBar|Hamburger"')>
         $if props['name'] == 'hamburger' and ctx.user:
             <img class="account__icon" src="https://archive.org/services/img/$(get_internet_archive_id(ctx.user.key))" alt="$('My account')">
+            $if get_counts_mode(mode='open') > 0 and is_privileged_user:
```

Comment:
```suggestion
            $if is_privileged_user:
```
No need to fetch the counts twice.

Reply:
I need that condition in order to check whether the bubble will be visible. And since it's cached for 1 minute, it won't make a call to the database. Please correct me if I am wrong.

Reply:
Yes.  I was trying to avoid all of the `get_counts_mode` calls, but it doesn't matter as much since the value will be cached.